resource-timing/resources/document-navigated.html

```diff
@@ -2,11 +2,20 @@
 <html>
 <head>
 <meta charset="utf-8" />
-</head>
-<body>
-    navigated document!
 <script>
+  function install_page_show_listener() {
+    window.addEventListener('pageshow', event => {
```

Comment:
Why do we need the pageshow event here if we plan to postmessage on load anyway?